app.py

```diff
@@ -21,10 +21,10 @@ def predict():
     #predict = prediction[:, 1]

```

Comment:
GPT summary of [036648](https://github.com/SampatSuthar/ml-demo/blob/cccc5e0aa50c29cda8434cc1a92463d295c7584b/app.py#0366480d84d3b97095bd42f20f2d7a550eadb09b) - [53fcbf](https://github.com/SampatSuthar/ml-demo/blob/0c97036f0373bc6c942bcdc96667a24a6851e90d/app.py#53fcbfb6eff29c32752a36cbdcba9f7463255086):
* Imported the openai library
* Updated the openai API key
* Changed the model from "davinci-003" to "text-davinci-003"
* Added a prompt for openai
* Added code to get the generated response from openai